src/notifications.c

```diff
@@ -222,6 +224,22 @@ int HashNotificationCallback(RedisModuleCtx *ctx, int type, const char *event,
       break;
   }
 
+  if (!strncmp(event, "json_", strlen("json_"))) {
+    if (!strncmp(event + JSON_LEN, "set", strlen("set")) ||
+        !strncmp(event + JSON_LEN, "del", strlen("del")) ||
+        !strncmp(event + JSON_LEN, "numincrby", strlen("numincrby")) ||
+        !strncmp(event + JSON_LEN, "nummultiby", strlen("nummultiby")) ||

```

Comment:
Should compare with `multby`  (event name is `json_multby`)

And handle missing `json_numpowby` ?